packages/reports/tests/acceptance/multi-datasource-test.js

```diff
@@ -186,13 +186,11 @@ module('Acceptance | multi-datasource report builder', function (hooks) {
     //check CSV export url
     await triggerEvent('.menu-trigger', 'mouseenter');
     await click($('.menu-content a:contains("CSV")')[0]);
-    assert
-      .dom('.export__download-link')
-      .hasAttribute(
-        'href',
-        'https://data2.naviapp.io/v1/data/inventory/day/container;show=desc/displayCurrency;show=desc/?dateTime=P3D%2Fcurrent&metrics=usedAmount%2Crevenue(currency%3DGIL)&filters=container%7Cid-in%5B%222%22%5D&having=usedAmount-gt%5B50%5D&format=csv',
-        'uses csv export from right datasource'
-      );
+    const csv = document.querySelector('.export__download-link')?.getAttribute('href') ?? '';
+    assert.ok(
+      /(?=.*?data2.naviapp.io)(?=.*format=csv)(?=.*dateTime=)(?=.*csv&filename=)/.test(csv),
+      'csv export url is generated for the report'
+    );
```

Comment:
do we test the filename itself or only that it exists?
to be more specific we can do `&filename=the_filename-\d\d\d\d-\d\d-\d\d` or whatever the date format is

also, would this work? (here and in the other asserts below)

```suggestion
    assert
      .dom('.export__download-link')
      .hasAttribute(
        'href',
        /(?=.*?data2.naviapp.io)(?=.*format=csv)(?=.*dateTime=)(?=.*csv&filename=)/,
        'csv export url is generated for the report'
      );
```

Reply:
Added date format in regex